src/ape_test/runners.py

```diff
@@ -1,42 +1,33 @@
 import pytest
-from _pytest.config import Config as PytestConfig
+from _pytest.config import Config, PytestPluginManager
 
 import ape
 from ape.logging import logger
-from ape.managers.chain import ChainManager
-from ape.managers.networks import NetworkManager
-from ape.managers.project import ProjectManager
+from ape.utils import ManagerAccessBase
 
 from .contextmanagers import RevertsContextManager
 
 
-class PytestApeRunner:
-    def __init__(
-        self,
-        config: PytestConfig,
-        project: ProjectManager,
-        networks: NetworkManager,
-        chain: ChainManager,
-    ):
-        self.config = config
-        self.project = project
-        self.networks = networks
-        self.chain = chain
+class PytestApeRunner(ManagerAccessBase):
+    def __init__(self):
         self._warned_for_missing_features = False
         ape.reverts = RevertsContextManager  # type: ignore
 
     @property
     def _network_choice(self) -> str:
         # The option the user providers via --network (or the default).
-        return self.config.getoption("network")
```

Comment:
I believe this was failing before because we were injecting the `config_manager` here?
The runner should get the `PytestConfig` the normal way probably?